user-guide/modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,886 @@
+
+In the directory where you saved `example.cpp`, issue the following command:
+
+Visual Studio Compiler:
+
+[source,none,subs="attributes+"]
+----
+cl /I C:\boost\include /link C:\boost\lib\libboost_json-vc71-mt-d-x86-1_34.lib example.cpp
+----
+
+GCC:
+
+[source,none,subs="attributes+"]
+----
+g++ -I /usr/local/include example.cpp -L /usr/local/libboost_json.a -o example
+----
+
+Clang:
+
+[source,none,subs="attributes+"]
+----
+clang++ -I /usr/local/include example.cpp -L /usr/local/libboost_json.a -o example
+----
+
+NOTE: Replace `C:\boost` or `/usr/local` with your Boost installation prefix if necessary.
+
+As boost:json[] is a compiled library, we need the linker option in our example.
+The name of the library file might vary according to your architecture and the options provided to `b2` while installing boost.
+Check the `lib` in your installation prefix.
+--
+========
+
+Search your computer for any JSON file, unless you have one you would like to use already.
+Record the full path to that file and run the example with:
+
+[tabs,sync-group-id=os]
+====
+Windows::
++
+--
+[source]
+----
+example "path/to/json/file.json"
+----
+--
+
+Linux::
++
+--
+[source]
+----
+./example "path/to/json/file.json"
+----
+--
+
+macOS::
++
+--
+[source]
+----
+./example "path/to/json/file.json"
+----
+--
+====
+
+Did you get the expected result?
+
+=== Summary
+
+Although the samples you have now built and run are quite simple, if you have got this far successfully, it means your build and installation and project linking are all working correctly.
+Great job!
+
+=== Next Steps
+
+You might like to scan the examples folders of some of the other libraries that you are interested in, and create and run projects to get them running.
+
+Once you are more experienced with Boost, you might like to build and install only those libraries you require.
+
+== Package Managers
+
+While we provide instructions for downloading and building the libraries from source, installing Boost using a package manager might simplify the process and ensure that all dependencies are installed correctly in some cases.
+In this guide, we will discuss two types of package managers: pass:[C++] package managers such as Conan and Vcpkg, and system package managers such as Homebrew and APT.
+
+While using a package manager can simplify the installation process, there are some potential downsides to consider, such as a lack of control over the installation process and potential version conflicts with other installed software.
+We will discuss these in more detail below.
+
+=== pass:[C++] Package Managers
+
+pass:[C++] package managers are a popular choice for managing dependencies in pass:[C++] projects.
+They provide a simple and consistent way to download, configure, and install libraries and their dependencies that's replicable on various platforms.
+However, these installation methods are not officially supported, which means the packages are not regularly tested by Boost authors.
```

Comment:
I think this discourages too much the use of vcpkg (which I think it's "almost official") and conan. I think these are good to use, the ones that get you really old software are the Linux ones.